css/base.css

```diff
@@ -35,13 +35,25 @@ body {
 }
 
 .main-nav ul {
-  display: none;
+  /*display: none;*/
 }
 
 .main-nav ul li {
```

Comment:
I would maybe put the nav bar CSS into the modules.css file. Totally up to you though.